CircleOfFifths/build.gradle

```diff
@@ -8,6 +8,7 @@ android {
     compileSdkVersion rootProject.compileSdkVersion
     buildToolsVersion rootProject.buildToolsVersion
     ndkVersion rootProject.ndkVersion
+    namespace = 'org.puredata.android.fifths'
```

Comment:
This, and similar changes in this PR, were necessary due to a breaking change in the [Android Gradle plugin](https://developer.android.com/build/releases/gradle-plugin).